de-lib/src/main/java/org/iplantc/de/desktop/client/presenter/DesktopPresenterImpl.java

```diff
@@ -456,6 +459,22 @@ public void onCollaboratorsClick() {
     }
 
     @Override
+    public void onPreferencesClick() {
+	final DesktopView.Presenter presenter = this;
+        preferencesDialogProvider.get(new AsyncCallback<PreferencesDialog>() {
+            @Override
+            public void onFailure(Throwable caught) {
+                ErrorHandler.post(caught);
+            }
+
+            @Override
+            public void onSuccess(final PreferencesDialog dialog) {
+                dialog.show(presenter, userSettings);
```

Comment:
Ah, right, in this context `this` is the AsyncCallback<PreferencesDialog> type.  So here, you could do `DesktopPresenterImpl.this` to get the correct "this" you want.